src/raymath.h

```diff
@@ -2580,4 +2580,319 @@ RMAPI void MatrixDecompose(Matrix mat, Vector3 *translation, Quaternion *rotatio
     }
 }
 
+
+// optional C++ math operators
+// define RAYLIB_DISABLE_VECTOR_OPERATORS to disable
+#if defined(__cplusplus) && !defined(RAYLIB_DISABLE_VECTOR_OPERATORS)
```

Comment:
The operators aren't just for vectors, and raymath can be used separately from raylib.
Probably this should just be named RAYMATH_DISABLE_OPERATORS or RAYMATH_NO_OPERATORS.